modules/serverless-kn-func-invoke.adoc

```diff
@@ -1,18 +1,21 @@
 // Module included in the following assemblies:
 //
 // * serverless/functions/serverless-functions-getting-started.adoc
-// * serverless/reference/kn-func-ref.adoc
+// * serverless/cli_tools/kn-func-ref.adoc
 
-:_content-type: REFERENCE
+:_content-type: PROCEDURE
 [id="serverless-kn-func-invoke_{context}"]
 = Invoking a deployed function with a test event
 
-You can use the `kn func invoke` CLI command to send a test request to invoke a function either locally or on your {product-title} cluster. This command can be used to test that a function is working and able to receive events correctly.
+You can use the `kn func invoke` CLI command to send a test request to invoke a function either locally or on your {product-title} cluster. This command can be used to test that a function is working and able to receive events correctly. Invoking a function locally is useful for a quick test during function development. Invoking a function on the cluster is useful for testing that is closer to the production environment.
```

Comment:
You can use this command to test if a function works and is able to receive events correctly.  [passive to active voice]

You can invoke:
- A function locally to perform a quick test during function development.
- A function on the cluster to test if it is closer to the production environment. [information mapping for more readability]

Reply:
@Amrita42 Used the first suggestion. We can't do information mapping here, unfortunately, because the abstracts are not supposed to contain lists anymore.